runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -66,7 +66,7 @@
      * A view of streams in the Corfu server instance.
      */
     @Getter(lazy = true)
-    private final MultiStreamView streamsView = new MultiStreamView(this);
+    private final MultiStreamView multiStreamsView = new MultiStreamView(this);
```

Comment:
this is getting out of hand. I have an idea, we can keep the classes named the name, but the fields shorter, sane names:

runtime.layout()
runtime.streams()
runtime.objects()
runtime.addressSpace()

-etc-